NSS-Day-Cohort-68/rare-client-rare-team-1
# Logout lmw

Closes https://github.com/NSS-Day-Cohort-68/rare-client-rare-team-1/issues/46

The purpose of this PR is to allow users to log out of the app.
 Slight bug fixes were added as per the conversation after lunch

File added:

/components/auth/Logout.js

Files changed:

/components/auth/Login.js
/components/tags/TagForm.js

Changes made:

Added 
Logout component to Logout.js to allow users to log out of the app.
Added the inability to click on the submit button if the text box is blank in Login.js and TagForm.js


 
Steps to test:

-On server side (rare-api) start the debugger
-Then on the client side, navigate to localhost:3000/login and log in to the app.
-Once logged in, open the dev tools in Chrome and select Application (may be in the 'more tabs' section). 
-Once there, on the left side should be a section call storage. Underneath storage is a sub-section called 'Local storage.' Click the triangle next to it and select http://localhost:3000. To the right of that should be a box for 'Key' and a box for 'Value.' Verify there is a key called 'rare-user.' This verifies that a user is logged in.
-Click the Logout button at the top of the page and verify that you're redirected to the Login screen.
-Check the dev tools to verify there is no longer a Key in the box.
-At login screen, verify that the login button is greyed out and cannot be pressed until something is written in the text box.
-Log in again.
-Navigate to Tags via the button on the navbar.
-Click on New Tag.
-Verify that the Add Tag button is greyed out and cannot be pressed until something is written in the text box.
